src/core/csv/reader_fread.cc

```diff
@@ -345,6 +347,7 @@ void FreadReader::detect_sep_and_qr() {
             << topNumFields << " fields using quote rule " << topQuoteRule;
       }
     }
+    if (!check_all_quoteRules && quoteRule == 1) break; else continue;
```

Comment:
i think `else  continue` is superfluous  here

Reply:
my bad, of course, it is redundant, thanks for that :)